app/models/categoryModels.js

```diff
@@ -9,5 +9,22 @@ const fetchAllCategories = () => {
     }
   });
 };
+const fetchAllReviews = () => {
+  const selectQuery = `
+        SELECT reviews.owner, title, reviews.review_id, category, review_img_url, reviews.created_at, reviews.votes, designer, COUNT(comment_id) AS comment_count 
+        FROM reviews 
+        JOIN comments 
```

Comment:
I think this should be a LEFT JOIN so that you don't omit reviews that have no comments.